### ISSUE TITLE
question about rvq_ce_loss

### ISSUE THREAD

**Issue by Shayne-Ada** [adelacvg/NS2VC#4]
Thank you for your code. I have a question.
In this line of code(https://github.com/adelacvg/NS2VC/blob/master/model.py#L506), do we need to add a softmax function to obtain the distribution?
<img width="710" alt="image" src="https://github.com/adelacvg/NS2VC/assets/24905716/707ce12f-1eac-41b0-8aa9-5440b3032d3d">

**Comment by adelacvg** [comment]
I believe that there is no need to add a softmax layer here. In fact, I have tried adding a softmax layer and the resulting loss was large and difficult to go down. The current implementation ensures that the ce_loss can go down normally during training. Feel free to modify the CE_loss module to achieve better results.

**Comment by Shayne-Ada** [comment]
Thank you for your explanation. I also forgot that F.cross_entropy includes softmax.
